cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -818,34 +893,36 @@ void esp_wifi_setup (esp_wifi_netdev_t* dev)
      */
     /* activate the Station and the SoftAP interface */
     result = esp_wifi_set_mode(WIFI_MODE_APSTA);
-#else /* MCU_ESP8266 */
+#else /* defined(MCU_ESP8266) && !defined(MODULE_ESP_WIFI_AP) */
```

Comment:
I think we don't need a comment here because it isn't a nested conditional.
```suggestion
#else
```
But if you wanna leave the comment, then update it to
```suggestion
#else /* defined(MCU_ESP8266) */
```